arm-visualstudio/2014-04-01-preview/swagger/Csm.json

```diff
@@ -0,0 +1,818 @@
+            "name": "extensionResourceName",
+            "description": "The name of the extension.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The response contains the extension resource details for the specified account.",
+            "schema": {
+              "$ref": "#/definitions/ExtensionResource"
+            }
+         },
+          "404": {
+            "description": "The specified extension has no plans defined."
+         }
+        },
+        "x-ms-examples": {
+            "getExtensionResource": {
+                "$ref": "../examples/GetExtensionResource.json"
+            }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Extensions"
+        ],
+        "summary": "Extensions_Update",
+        "description": "Updates an existing extension registration for the Visual Studio Team Services account.",
+        "operationId": "Extensions_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "An object containing additional information related to the extension request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ExtensionResourceRequest"
+            }
+          },
+          {
+            "in": "path",
+            "name": "accountResourceName",
+            "description": "The name of the Visual Studio Team Services account resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "extensionResourceName",
+            "description": "The name of the extension.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The extension resource was updated for the specified account.",
+            "schema": {
+              "$ref": "#/definitions/ExtensionResource"
+            }
+         }
+        },
+        "x-ms-examples": {
+            "updateExtensionResource": {
+                "$ref": "../examples/UpdateExtensionResource.json"
+            }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/microsoft.visualstudio/account/{resourceName}": {
+      "put": {
+        "tags": [
+          "Accounts"
+        ],
+        "summary": "Accounts_Create",
+        "description": "Creates or updates a Visual Studio Team Services account resource.",
+        "operationId": "Accounts_Create",
```

Comment:
From comments looks like service support updating an account via PUT if so then correct operation id is `Accounts_CreateOrUpdate`